diff --git a/node_modules/react-native-reanimated/android/CMakeLists.txt b/node_modules/react-native-reanimated/android/CMakeLists.txt
index d4731e6..6aa1c91 100644
--- a/node_modules/react-native-reanimated/android/CMakeLists.txt
+++ b/node_modules/react-native-reanimated/android/CMakeLists.txt
@@ -74,20 +74,8 @@ target_include_directories(
 
 set_target_properties(reanimated PROPERTIES LINKER_LANGUAGE CXX)
 
-if(${CMAKE_BUILD_TYPE} MATCHES "Debug")
-  set(BUILD_TYPE "debug")
-else()
-  set(BUILD_TYPE "release")
-endif()
-
-add_library(worklets SHARED IMPORTED)
-
-set_target_properties(
-  worklets
-  PROPERTIES
-    IMPORTED_LOCATION
-    "${REACT_NATIVE_WORKLETS_DIR}/android/build/intermediates/cmake/${BUILD_TYPE}/obj/${ANDROID_ABI}/libworklets.so"
-)
+# Worklets 0.7.2 publishes its library through Prefab.
+find_package(react-native-worklets REQUIRED CONFIG)
 
 set_target_properties(reanimated PROPERTIES LINKER_LANGUAGE CXX)
 # remove dead code sections
@@ -108,4 +96,4 @@ target_link_libraries(
   ReactAndroid::jsi
   fbjni::fbjni
   android
-  worklets)
+  react-native-worklets::worklets)
